src/devices/lightbulbs.ts

```diff
@@ -493,7 +516,7 @@ export class lightBulb {
     this.platform.log.debug('%s %s - Set On: %s', this.device.model, this.accessory.displayName, value);
 
     this.On = value;
-    this.doUpdate.next();
+    this.pushOnChanges();
```

Comment:
I see your going right to pushOnChanges, is there a reason why you are eliminated the need to look at `this.doUpdate.next();`? 

Reason I ask is because this not sure `this.doUpdate.next();` is needed anymore if we going to right to `this.pushOnChanges();`

Reply:
doUpdate.next() will update every value, regardless if it actually updated or not. Is there even a case where this behavior is needed? You are right regarding the changes from this PR, the doUpdate() is then not needed anymore.